scripts/dev_setup.py

```diff
@@ -1,31 +1,41 @@
-#!/usr/bin/python
+#!/usr/bin/env python
 
 #---------------------------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 #---------------------------------------------------------------------------------------------
 
 from __future__ import print_function
+
 import sys
 import os
 from subprocess import check_call, CalledProcessError
 
 root_dir = os.path.abspath(os.path.join(os.path.abspath(__file__), '..', '..'))
 
+# color code used in the print statement:
+#   green \033[92m
+#   blue  \033[94m
+#   end   \033[0m
+
 def exec_command(command):
     try:
-        print('Executing: ' + command)
+        print('\033[92mExecuting: ' + command + '\033[0m')
         check_call(command.split(), cwd=root_dir)
+        print('\n')
```

Comment:
Use `print()`.
`print('\n')` is the same as `print()`:
see https://docs.python.org/3/library/functions.html#print